### PR TITLE
Fix table view delta-T column.

### PR DESCRIPTION
**Pull request by hjoliver**
The mean delta-T (mean elapsed run time) column in the table view was not getting populated.

Also tidy the table header a bit.

This is a small change with no associated Issue.

Screenshot from 2022-10-14 17-50-27

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).

- [ ] Appropriate tests are included (unit and/or functional).
- [x] Already covered by existing tests.
- [ ] Does not need tests (why?).

- [ ] Appropriate change log entry included.
- [ ] No change log entry required (why? e.g. invisible to users).

- [ ] I have opened a documentation PR at cylc/cylc-doc/pull/XXXX.
- [x] No documentation update required.

### CODE REVIEW

File: src/components/cylc/table/Table.vue [hunk 1]
```diff
@@ -233,7 +239,7 @@ export default {
       headers: [
         {
           text: 'Task',
-          value: 'node.name',
+          value: 'node_name',
```
Comment (oliver-sanders) [inline src/components/cylc/table/Table.vue:242]: Where is `node_name` defined?

File: src/components/cylc/table/Table.vue [hunk 2]
```diff
@@ -110,6 +110,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
             item-key="id"
             show-expand
           >
+
+            <!-- eslint-disable-next-line -->
+            <template v-slot:header.node_task_meanElapsedTime="{ header }">
+              <span v-html="header.text"></span>
+            </template>
+
```
Comment (oliver-sanders) [inline src/components/cylc/table/Table.vue:118]: The view seems to work the same with or without this template.

```suggestion
```

File: src/components/cylc/table/Table.vue [hunk 3]
```diff
@@ -243,42 +249,42 @@ export default {
         },
         {
           text: 'Cycle Point',
-          value: 'node.cyclePoint',
+          value: 'node_cyclePoint',
```
Comment (oliver-sanders) [inline src/components/cylc/table/Table.vue:252]: I don't think these changes work, it breaks sorting the table columns.

File: src/components/cylc/table/Table.vue [hunk 4]
```diff
@@ -243,42 +249,42 @@ export default {
...
-          text: 'T-finish',
-          value: 'latestJob.finishedTime',
+          text: 'T-Finish',
+          value: 'latestJob_finishedTime',
           sort: datetimeComparator
         },
         {
-          text: 'dT-mean',
-          value: 'meanElapsedTime',
+          text: '&Delta;T-Mean',
```
Comment (oliver-sanders) [inline src/components/cylc/table/Table.vue:286]: `&Delta;` doesn't get expanded into a delta symbol. If you wanted the unicode delta symbol here, I think you can write it in plain text, no escape needed, the web runs on UTF-8.

**Review by oliver-sanders — CHANGES_REQUESTED** [review]
The "T-Mean" column works but this breaks column sorting (but not the tests, yay).

Raised a PR: https://github.com/hjoliver/cylc-ui/pull/60

**Comment by oliver-sanders** [comment]
I've applied the fix from this PR onto master and added some tests in #1164.

**Comment by MetRonnie** [comment]
I'm closing as superseded by #1164

**Comment by hjoliver** [comment]
Thanks for taking this over in my absence @oliver-sanders

File: src/components/cylc/table/Table.vue (hunk 4 above)
Comment (hjoliver) [reply src/components/cylc/table/Table.vue:286]: BTW this did render a Delta symbol - see the screenshot on this issue. The reason was the `v-html` template you commented on above https://github.com/cylc/cylc-ui/pull/1112/files#r1045695288 ... However, you're right about just using UTF-8!

**Pull request closed without merging**